resotocore/resotocore/graph_manager/graph_manager.py

```diff
@@ -85,20 +86,20 @@ async def _on_config_updated(self, config_id: str, data: Json) -> None:
             await self.__setup_cleanup_old_snapshots_worker(snapshots_config)
 
             # cancel all existing snapshot jobs
-            existing_jobs = [job for job in await self.task_handler.list_jobs() if job.id.startswith("snapshot-")]
+            existing_jobs = [job for job in await self.task_handler.list_jobs() if job.id.startswith(job_prefix)]
             for job in existing_jobs:
                 await self.task_handler.delete_job(job.id)
 
             # schedule new snapshot jobs for the current graph
             for label, schedule in snapshots_config.snapshots.items():
```

Comment:
Is the label being cleaned? Consider the label being user-input and only allow [a-zA-Z0-9] as value.

Reply:
That's already the case.